changelog/issue-2031.md

```diff
@@ -0,0 +1,4 @@
+level: minor
+reference: issue 2031
+---
+Taskcluster UI pickers now feature a time component, so for instance, users can now pick a particular time (hour:minute) on a particular day to set as the expiration time.
```

Comment:
Taskcluster UI revamped the date picker component to allow selecting the hour and the minute in addition to the date.

Reply:
Done